cengage/react-magma
# feat(Modal): Addition of a headerRef prop to access the Modal header when needed.

Issue: # [1308](https://github.com/orgs/cengage/projects/1/views/7?pane=issue&itemId=66247542)

## What I did
- Addition of a `headerProp` for access to the Modal header, useful for keyboard navigation.

## Checklist 
- [N/A] New and existing unit tests pass locally with my changes
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- In Docs and Storybook go to "Header Ref" example.
- Open Modal, click on "Header Ref" button.
- Ensure header gets custom styling applied to the H1.
